runtime/doc/options.txt

```diff
@@ -1940,6 +1940,21 @@ A jump table for the options with a short description can be found at |Q_op|.
 	This option cannot be set from a |modeline| or in the |sandbox|, for
 	security reasons.
 
+						*'completepathslash'* *'cps'*
+'completepathslash' 'cps'	boolean	(default off)
+			local to buffer
+			{not in Vi}
+			{not available when compiled without the |+windows|
```

Comment:
`|+windows|` doesn't mean MS-Windows. This should be `{only for MS-Windows}` (like `'shellslash'`).